src/mixins/nr-affiliation-mixin.ts

```diff
@@ -188,8 +188,13 @@ export class NrAffiliationMixin extends Mixins(CommonMixin) {
       // show spinner since this is a network call
       this.$root.$emit('showSpinner', true)
       const accountId = +JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))?.id || 0
-      const businessId = await this.createBusinessIA(accountId, legalType)
-      this.goToEntityDashboard(businessId)
+      // incorp requires user login, ignore if user is not logged in and return to NR page
+      if (accountId === 0) {
```

Comment:
It seems kludgy to me to catch this scenario here (which is why it's hard to explain with code comments).

This function is called from 2 places, right?
1. The search page (which only calls this if authenticated, so NOT the scenario you're protecting from).
2. When the app is reloaded (where we expect the user to have logged in if we find NR_DATA in session storage).

Did you consider skipping over the code in App.vue if NR_DATA exists but the user is not authenticated? It seems to me this is a much more direct solution to the problem.